activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Make `ActiveSupport::Cache::Strategy::LocalCache::Middleware` updatable.
```

Comment:
I think the changelog entry could be a bit more descriptive - because we're really making it possible to update the LocalCache rather than making the Middleware updatable. Meaning that I wouldn't know what is updatable by reading this changelog entry.